src/Nethermind/Nethermind.Config/BlocksConfig.cs

```diff
@@ -58,5 +58,7 @@ public byte[] GetExtraDataBytes()
         public string GasToken { get => GasTokenTicker; set => GasTokenTicker = value; }
 
         public static string GasTokenTicker { get; set; } = "ETH";
+
+        public long BlockProductionMaxTxKilobytes { get; set; } = 9728;
```

Comment:
are you sure it is correct value? how much CL can have on worst-case?

Reply:
Docs say 127kB in worse case; but not sure they are outdated so increased to 512kB (as also EL payload framing)

1024*10-512 = 9728